db/python/tables/sequence.py

```diff
@@ -377,9 +377,9 @@ async def update_sequence(
             if technology:
                 updaters.append('technology = :technology')
                 fields['technology'] = technology.value
-            if sequence_type:
-                updaters.append('sequence_type = :type')
-                fields['sequence_type'] = sequence_type.value
+            if type:
+                updaters.append('type = :type')
+                fields['type'] = type.value
```

Comment:
These two lines are the only ones that need to change, redefining the builtin type is a little risky so would prefer to stray away from doing that.

Reply:
I think that's totally fair. I've reverted all the other changes except for the above lines, now the updater mostly refers to the field as sequence_type.